### PR TITLE
[TDP survey] Display user initials only in print media

### PR DESCRIPTION
**Pull request by mrclay**
Hide user-given initials except for print media.

---

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: cfgov/unprocessed/apps/teachers-digital-platform/css/pages/survey-results.less [hunk 1]
```diff
@@ -1,3 +1,14 @@
+@media not print {
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/teachers-digital-platform/css/pages/survey-results.less:1]: Just a thought but should you maybe make it `display: none` by default and display it with `@media print`? I haven't looked into the support for `@media` versus `@media not`, but negating the hiding seems safer for any browser that may not support negated media queries.

Comment (mrclay) [reply cfgov/unprocessed/apps/teachers-digital-platform/css/pages/survey-results.less]: Updated.

**Review by higs4281 — COMMENTED** [review]
The "share results" button pops a dialog asking for initials that won't be used. Shouldn't the initials entry box only appear  when you click on "Print results"?

**Comment by mrclay** [comment]
@higs4281 The initials are still encoded in the shared URL to be visible to the teacher (when printed).

**Comment by mrclay** [comment]
To be clear, this change still allows setting 4 arbitrary characters, but the "fun share factor" is considerably lower since the receiver won't see them unless they also print.

**Comment by higs4281** [comment]
It's great that the initials aren't displayed, but why ask for initials at all for the shareable link if neither student nor teacher will see them in the linked page?

**Comment by mrclay** [comment]
Presumably some teachers who want to print will appreciate having the initials on there (or to preview just in case they forget who sent it). Is this functionality still a problem?

**Comment by higs4281** [comment]
Oh, i see. The teacher gets a URL that shows no initial, but if printed the initials show up.  

Sorry, I was getting confused testing locally because some of the steps have www hard-coded, and I was being taken to the live site instead of localhost:8000.

**Review by higs4281 — APPROVED** [review]

**Pull request merged**